app/tools/android/make_apk.py

```diff
@@ -590,7 +590,11 @@ def main(argv):
       'They are used for various settings for applications through '
       'command line options.')
   info = ('The version name of the application. '
-          'For example, --app-version=1.0.0')
+          'For example, --app-version=1.0.0 '
+          'If the field length of app version is greater than 3, '
+          '--app-versionCode or --app-versionCodeBase should be specified '
+          'in command line. '
+          'For example, --app-version=1.0.0.1 --app-versionCodeBase=24')
```

Comment:
A little confused by "'For example, --app-version=1.0.0.1 --app-versionCodeBase=24')".
should it be: --app-version=24.0.0.1 --app-versionCodeBase=24?